app/auth-portal/src/content/tutorial/08-asset_authoring.md

```diff
@@ -0,0 +1,442 @@
+      message: component.properties.resource?.message,
+    };
+  }
+
+  const child = await siExec.waitUntilEnd("aws", [
+    "ec2",
+    "describe-volumes",
+    "--volume-ids",
+    resource.VolumeId,
+    "--region",
+    component.properties.domain.region,
+  ]);
+
+  if (child.exitCode !== 0) {
+    console.log(`Volume ID: ${resource.VolumeId}`);
+    console.error(child.stderr);
+    return {
+      payload: resource,
+      status: "error",
+      message: `AWS CLI 2 "aws ec2 describe-volumes" returned non zero exit code (${child.exitCode})`,
+    };
+  }
+
+  return { payload: JSON.parse(child.stdout).Volumes[0], status: "ok" };
+}
+```
+
+### Let's test it
+
+So we have authored and attached all the functions required to manage our infrastructure. Now we can test it. Let's open the
+System Initiative modelling UI and drag an EBS Volume onto the canvas:
+
+![unqualified.png](/tutorial-img/08-asset-authoring/unqualified.png)
+
+We will see that it is initially `unqualified` (if you click on the hex with the red X, you will see why it's so). Let's
+update the attributes of the asset:
+
+![component-attributes.png](/tutorial-img/08-asset-authoring/component-attributes.png)
+
+```text
+availabilityZone: us-east-2a
+region: us-east-2
+size: 100
+iops: 3000
+volumeType: gp3
+```
+
+The function will execute, and we can see that the hex goes to a green check mark. We can use the `Apply Changes` button
+to deploy this EBS Volume, and we should see a successful deployment.
+
+![successful-deployment.png](/tutorial-img/08-asset-authoring/successful-deployment.png)
+
+The solid green hex means a successful deployment! Now we can contribute the asset so that others can use it.
+
+### Contribution time
+
+If we go back to the `Customize` screen, the Assets tab should be selected and we should see a `Contribute` button. That
+can be used to package up the Asset and share it with System Initiative who will test it and make it part of the available
+asset collection for the community. Let's go ahead and do that:
+
+![contribute-button.png](/tutorial-img/08-asset-authoring/contribute-button.png)
+
+If we are happy, we can whack the button:
+
+![contribution-sent.png](/tutorial-img/08-asset-authoring/contribution-sent.png)
+
+We love our community and are always happy to accept contributions for our assets!!
+
+### Summary
+
+In this post, we've taken a look at how to create a schema definition and all the functions that go with it to create
+an asset. We were able to verify that the asset works and even contributed that asset back to System Initiative to use for
+the community. We are excited to see what assets that you get to make and even more excited to test all the assets that
+you want to share with the community. If you have any questions, issues or feedback about asset authoring, we invite you to join
+us on [Discord](https://discord.com/invite/system-init) and talk to us there. You can also message me there directly at `SI_Stack72`. 
```

Comment:
Maybe say something about how we are kicking off asset authoring, and there is tons of meaningful work to be done building the assets and making the asset building experience better. If folks want to work with us on this, we would love to help them figure things out, get unstuck, find solutions. Come to the Asset Authoring channel in discord to talk about what you want to build and we'll do it together.

Reply:
This entire section will be removed :) It's left there by mistake from the blog post